tests/semanticdb/metac.expect

```diff
@@ -2022,9 +2022,6 @@ Occurrences:
 [5:4..5:8): List -> scala/package.List.
 [5:9..5:10): x -> local0
 
-Synthetics:
-[5:4..5:8):List => *.apply[Int]
-
```

Comment:
I don't understand what this file is, but I wonder whether this change is perhaps unrelated to the current PR.

Reply:
It can be ignored. I think it was part of the first commit.